src/EnergyPlus/UnitarySystem.cc

```diff
@@ -16066,12 +16070,13 @@ namespace UnitarySystems {
         }
 
         if (this->m_SuppCoilExists) {
-            if (this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingElectric) {
+            if (this->m_SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingElectric ||
+                this->m_SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingElectric_MultiStage) {
```

Comment:
Fixed incorrect variable use here. The heating coil type Number used should have been for supplemental heater instead of the main heating coil type Number. This would work only when the main and the supplemental heating coil are the same heating coil type.